frontend/src/views/GuideIdentificationFirearm/GuideIdentificationFirearm.vue

```diff
@@ -33,8 +33,10 @@ const currentStep = computed({
   },
 })
 
+const ALARM_GUNS_TYPOLOGIES = ['pistolet_semi_auto_moderne', 'revolver']
+
 const steps = computed(() => (resultTree[resultStore.typology].isDummyTypology || confidenceLevel.value !== 'low')
```

Comment:
a bit hard to read because very condensed ^^ please add a comment
"
if low confidence level -> 1 step ('Résultat final')  
else -> min 2 steps ('Typologie de l\'arme', 'Résultat final')  
if typo can be dummy -> add 2 steps ('Compléments', 'Typologie de munitions')  
if typo can be alarm -> add 1 step ('Identification d\'une arme d\'alarme') (all alarm gun typologies can be dummies)
"